src/shared-components/segmentedControl/style.ts

```diff
@@ -0,0 +1,78 @@
+import styled from '@emotion/styled';
+import { buttonReset } from 'src/utils/styles/buttonReset';
+
+import { COLORS, SPACER, BOX_SHADOWS } from '../../constants';
+import { style as TYPOGRAPHY_STYLE } from '../typography';
+
+type SegmentItemProps = {
+  active: boolean;
+  key: number;
+  onClick: () => void;
+  width: number;
+};
+
+type IndicatorProps = {
+  width: number;
+  transform: string;
+};
+
+export const SegmentsContainer = styled.div`
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  border-radius: 80px;
+  background-color: ${COLORS.purple10};
+  padding: ${SPACER.xsmall};
+  border: 4px solid ${COLORS.purple10};
+`;
+
+export const SegmentItem = styled.button<SegmentItemProps>`
+  ${buttonReset};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  border: none;
+  ${TYPOGRAPHY_STYLE.caption};
+  color: ${COLORS.purple70};
+  border-radius: 80px;
+  background-color: ${COLORS.purple10};
+  width: ${({ width }) => `${width}%`};
+  &:focus {
+    outline: none;
+    box-shadow: 0px 0px 0px 2px ${COLORS.primary};
+    z-index: 3;
```

Comment:
We have a z-index constants file that might be worth using here, just to avoid hard-coded values: `src/constants/zScale/index.ts`